test/test_open.py

```diff
@@ -12,6 +13,23 @@
 FILENAME = Path('one')
 
 
+# Windows OS helper for checking admin privileges
+def is_windows_admin():
+    """Check if the script is running in a terminal with admin privileges on Windows"""
+    if os.name == 'nt':
+        try:
+            return ctypes.windll.shell32.IsUserAnAdmin()
+        except Exception:
+            return False
+
+
+IS_WINDOWS_USER = os.name == 'nt' and not is_windows_admin()
```

Comment:
Move the constants to the top where they'll be happy with their constant buddies. 🍳

Reply:
It needs the `is_windows_admin()` function. It needs to be defined before I can use it.